src/app/pages/tags/tag-assign/tag-assign.component.ts

```diff
@@ -77,12 +77,10 @@ export class TagAssignComponent implements OnInit {
     this.userID = this.formGroup.controls['userID'];
     this.default = this.formGroup.controls['default'];
     this.default.setValue(false);
-    if (!this.canListUsers) {
```

Comment:
Removed because siteOwner can list users but cannot assign/create badge for other users in authorizationsDefinition